cluster/azure/util.sh

```diff
@@ -34,6 +34,9 @@ KUBE_CONFIG_FILE="${KUBE_CONFIG_FILE:-"${DIR}/config-default.sh"}"
 source "${KUBE_CONFIG_FILE}"
 source "${KUBE_ROOT}/cluster/common.sh"
 
+AZKUBE_VERSION="v0.0.4"
+REGISTER_MASTER_KUBELET="true"
```

Comment:
(Just to clarify -- this usage is in `cluster/` as well. The `hack/` script isn't used in the mainline for the cluster bring-up. It's just something I'm using while iterating on a new cloudprovider plugin and thought other hyperkube-users might use as well.)